libbeat/tests/system/test_base.py

```diff
@@ -39,6 +39,22 @@ def test_invalid_config(self):
         assert exit_code == 1
         assert self.log_contains("error loading config file") is True
 
+    def test_invalid_config_cli_param(self):
+        """
+        Checks CLI overwrite actually overwrites some config variable by
+        writing an invalid value.
+        """
+
+        self.render_config_template(
+            console={"pretty": "false"}
+        )
+
+        exit_code = self.run_beat(
```

Comment:
what exactly would you look for? Just relying on exit_code doesn't tell me if configuration was overwritten correctly.

Would love to introduce another test showing `-E` did apply config correctly. Maybe introduce some special config setting for printing a custom message (like `mockbeat.echo`) ?